catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/util/CrossPlatformFilePathEvaluator.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+
+package org.codice.ddf.commands.util;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import ddf.security.common.audit.SecurityLogger;
+
+/*
+ * Handles converting given Windows file paths (whose backslashes get escaped) into something the
+ * commands can handle.
+ */
+public class CrossPlatformFilePathEvaluator {
+
+    public static File handlePath(String path) {
+        //return java can find the file using the path
+        File file = new File(path);
+        if (file.exists()) {
+            return file;
+        }
+
+        //look for drive letter
```

Comment:
❗️ Let's protect this code so it only runs if Windows (FilenameUtils.isSystemWindows())